scripts/githooks/check-file-format.sh

```diff
@@ -75,8 +68,32 @@ function is-arg-true() {
 
 # ==============================================================================
 
+check=${check:-working-tree-changes}
```

Comment:
Could we move the `case $check in` section inside the `main` function? Perhaps, we could do the same with the `dry_run` flag but keep the `VERBOSE` flag where it is, as it's used for debugging.

Reply:
Sure, that's straightforward.